.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
   - 8.11.1
+script: npm run lint && npm test
 before_script:
   - npm install -g grunt-cli
+cache:
+  directories:
+    - "node_modules"
```

Comment:
Not really related to this PR, but it caches the `node_modules` directory between CI runs (updating it where necessary). It should speed things up by a lot.